examples/run/cuda/seq_example_cuda.cpp

```diff
@@ -350,12 +372,33 @@ int seq_run(const traccc::opts::detector& detector_opts,
                                     vecmem::get_data(params));
                 }
 
+                // CUDA
+                {
+                    traccc::performance::timer timer{
+                        "Ambiguity resolution (cuda)", elapsedTimes};
+                    res_track_candidates_buffer =
+                        resolution_alg_cuda(track_candidates_buffer);
+                }
+
+                // CPU
+                auto track_candidates_cuda_tmp =
+                    copy_track_candidates(track_candidates_buffer);
+
+                if (accelerator_opts.compare_with_cpu) {
+                    traccc::performance::timer timer{
+                        "Ambiguity resolution (cpu)", elapsedTimes};
+                    res_track_candidates = resolution_alg_cpu(
+                        traccc::get_data(track_candidates_cuda_tmp));
+                    // resolution_alg_cpu(traccc::get_data(track_candidates));
```

Comment:
I will uncomment this before the PR is merged. Currently I am using cuda CKF output for the comparison

Reply:
Don't forget about undoing/fixing this!